sw-emulator/app/src/main.rs

```diff
@@ -155,6 +170,9 @@ fn main() -> io::Result<()> {
         .get_matches();
 
     let args_rom = args.get_one::<PathBuf>("rom").unwrap();
+    let args_recovery_fw = args.get_one::<PathBuf>("recovery-image-fw");
+    let _args_recovery_image_manifest= args.get_one::<PathBuf>("recovery-image-manifest"); // TODO hook up to RRI
+    let _args_recovery_image_mcurt= args.get_one::<PathBuf>("recovery-image-mcurt"); // TODO hook up to RRI
     let args_current_fw = args.get_one::<PathBuf>("firmware");
```

Comment:
let args_current_fw = args.get_one::<PathBuf>("firmware"); 

This should be removed as recovery-image-fw is the firmware. Also, rename the update-firmware to recover-image-fw-update